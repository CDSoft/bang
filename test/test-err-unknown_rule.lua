generator(false)

rule "foo" {}
build "bar" { "baz", "file" } -- [test/test-err-unknown_rule.lua:4] ERROR: baz: unknown rule
