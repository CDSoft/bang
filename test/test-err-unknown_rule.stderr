[test/test-err-unknown_rule.lua:4] ERROR: baz: unknown rule
